install_scripts/install_udp_rx.sh

```diff
@@ -0,0 +1,72 @@
+# create a service user
+echo "Creating User"
+useradd -r udprx
+# create /etc/udp_rx directory
+echo "Creating udp_rx directories"
+mkdir -p /etc/udp_rx
+chmod 755 /etc/udp_rx
+# move the default portlist to that directory
+echo "Moving portslist and setting permissions"
+cp portslist /etc/udp_rx/
+chown root /etc/udp_rx/portslist
+chgrp root /etc/udp_rx/portslist
+chmod 644 /etc/udp_rx/portslist
+# move the default config file to that directory
+echo "Moving configuration files"
+cp udp_rx_conf.json /etc/udp_rx/
+chown root /etc/udp_rx/udp_rx_conf.json
+chgrp root /etc/udp_rx/udp_rx_conf.json
+chmod 644 /etc/udp_rx/udp_rx_conf.json
+# if the keys and the CA cert exist, move them
+# key 
+if [ -e udp_rx.key ]
+then
+    echo "Found udp_rx.key, copying to /etc/udp_rx"
+    cp udp_rx.key /etc/udp_rx/
+    chown root /etc/udp_rx/udp_rx.key
+    chgrp root /etc/udp_rx/udp_rx.key
+    chmod 644 /etc/udp_rx/udp_rx.key
+else
+    echo "WARNING: Couldn't find udp_rx.key, you will need to set the location in the configuration file and/or move the file yourself"
+fi
+# cert
+if [ -e udp_rx.cert ]
+then
+    echo "Found udp_rx.cert, copying to /etc/udp_rx"
+    cp udp_rx.cert /etc/udp_rx/
+    chown root /etc/udp_rx/udp_rx.cert
+    chgrp root /etc/udp_rx/udp_rx.cert
+    chmod 644 /etc/udp_rx/udp_rx.cert
+else
+    echo "WARNING: Couldn't find udp_rx.cert, you will need to set the location in the configuration file and/or move the file yourself"
+fi
+# ca cert
+if [ -e ca.cert.pem ]
+then
+    echo "Found ca.cert.pem, copying to /etc/udp_rx"
+    cp ca.cert.pem /etc/udp_rx/
+    chown root /etc/udp_rx/ca.cert.pem
+    chgrp root /etc/udp_rx/ca.cert.pem
+    chmod 644 /etc/udp_rx/ca.cert.pem
```

Comment:
I just want to confirm that it's safe for the world to know where we are storing stuff, right? I know they would still have to get into our system to do anything about it, so this info is probably not a big deal.

Reply:
correct, this is just the default path for the installation, and ONLY if they are put next to the installer at runtime